ui/src/main/resources/FileManagerCode/Macros.xml

```diff
@@ -418,14 +418,15 @@
       ## Add the file object.
       #set ($discard = $fileDoc.newObject('FileManagerCode.FileClass').set('description', "$!request.description"))
       ## Add the attachment.
-      ## FIXME: Get the file content as an input stream instead of a byte array, but for this we need to expose
-      ## getFileItemInputStream in the public API of the fileupload plugin (or use a script service).
-      #set ($bytes = $xwiki.fileupload.getFileItemData('filepath'))
-      #if (!$bytes)
-        ## Empty file (unfortunately getFileItemData returns null instead of an empty byte array).
-        #set ($bytes = [])
+      ## Find the attachment among the fields of this form-submission.
+      #set($fileUploadFiles = $xwiki.fileupload.getFileItems())
+      #set($inputStream = "")
+      #foreach($fileItem in $fileUploadFiles)
+        #if('filepath'==$fileItem.getFieldName())
```

Comment:
The code style still doesn't match the rest of the file. Check for instance the ``#if`` below. For me it's important to have a consistent code style.

Reply:
Do we have a tool for this?
I assume I might be breaking style elsewhere.
thanks in advance.
paul

Reply:
I believe that this is fixed in b5ff50ef2811e6a550ce1b7388744b7a70baa3d7.
Thanks to check.
Paul

Reply:
Is it?

Reply:
Yes, thanks!